tests/models/test_anthropic_bedrock_count_tokens.py

```
@@ -0,0 +1,242 @@
+
+async def test_anthropic_bedrock_count_tokens_unexpected_response(env: TestEnv):
+    """Pins the defensive `UnexpectedModelBehavior` branch for a malformed Bedrock response.
+
+    Mocks `client.post` to return a body without `inputTokens` — a shape no real Bedrock
+    CountTokens endpoint returns, so it can't be exercised through a VCR recording.
+    """
+    from anthropic import AsyncAnthropicBedrock
+
+    bedrock_client = AsyncAnthropicBedrock(
+        aws_access_key='test-access-key',
+        aws_secret_key='test-secret-key',
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to argument: "aws_secret_key"

(S106)